Packs/MicrosoftGraphUser/Integrations/MicrosoftGraphUser/MicrosoftGraphUser_description.md

```diff
@@ -1,30 +1,32 @@
+To use this integration, you need to configure authentication. There are three authentication methods available:  
+- Cortex XSOAR Azure app 
+- Self-Deployed Azure app  
+- Azure Managed Identities Authentication
```

Comment:
```suggestion
- [Cortex XSOAR Azure app]{#cortex-xsoar-azure-app)
- [Self-Deployed Azure app](#self-deployed-azure-app)  
- [Azure Managed Identities Authentication](#azure-managed-identities-authentication)
```

Reply:
Hi @ShirleyDenkberg, thanks for the review.
When clicking on the link in the description.md file, this window opens - https://server_url/#/cortex-xsoar-azure-app, so it is useless.

Reply:
bummer.  Works well in a readme file